app/config/security.yml

```diff
@@ -192,6 +192,7 @@ security:
             - ROLE_APP_ADMIN_MOOC_MOOC_ALL
             - ROLE_APP_ADMIN_MOOC_CHAPTER_ALL
             - ROLE_APP_ADMIN_MOOC_VIDEO_ALL
+            - ROLE_APP_ADMIN_MOOC_QUIZZ_ALL
```

Comment:
This role is incorrect.

You should add the role  `ROLE_APP_ADMIN_MOOC_ELEMENT_ALL` instead.
And remove `ROLE_APP_ADMIN_MOOC_VIDEO_ALL` and `ROLE_APP_ADMIN_MOOC_QUIZZ_ALL`

Reply:
yes